snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: retdec
+version: "8e65c05"
+summary: RetDec
+description: |
+  RetDec is a retargetable machine-code decompiler based on LLVM.
+  The decompiler is not limited to any particular target architecture,
+  operating system, or executable file format.
+  Supported file formats are ELF, PE, Mach-O, COFF, AR (archive), Intel HEX, and raw machine code.
+  Supported architectures (32b only) are Intel x86, ARM, MIPS, PIC32, and PowerPC.
+
+confinement: strict
+
+apps:
+  retdec:
+    command: bin/retdec-decompiler.py
+
+parts:
+  retdec:
+    plugin: cmake
+    configflags:
+      - -DCMAKE_INSTALL_PREFIX=/
+    source-type: git
+    source: https://github.com/avast-tl/retdec.git
```

Comment:
Were you able to build a snap package by using the submitted `snapcraft.yml` file? I was not yet able to try it, but what seems fishy to me is that RetDec has a lot of dependencies (see e.g. [Dockerfile](https://github.com/avast-tl/retdec/blob/master/Dockerfile)), and none of them are listed in the `snapcraft.yml` file.

Reply:
Yes, I have built the snap on my system. It's uploaded to the snap store  at:
https://snapcraft.io/retdec

I tested the snap build on my system, which did have all the build dependencies. But the point of a snap is that the dependencies get built in so it should work on other systems the same way.